clang-tools-extra/modularize/ModularizeUtilities.cpp

```diff
@@ -435,9 +435,8 @@ static std::string replaceDotDot(StringRef Path) {
   llvm::sys::path::const_iterator B = llvm::sys::path::begin(Path),
     E = llvm::sys::path::end(Path);
   while (B != E) {
-    if (B->compare(".") == 0) {
-    }
-    else if (B->compare("..") == 0)
+    if (*B == ".") {
+    } else if (*B == "..")
       llvm::sys::path::remove_filename(Buffer);
     else
```

Comment:
Refactor this to `else if (*B != ".")`

Reply:
Thank you for reviewing the patch!  Fixed.